react-app/src/components/ProposalDetailScreen/ProposalDescription.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback } from "react";
+import MDEditor from "@uiw/react-md-editor";
+import cn from "classnames";
+import { toast } from "react-toastify";
+import Paper from "../common/Paper/Paper";
+import IconButton from "../common/Buttons/IconButton";
+import { IconType } from "../common/Icons/Icons";
+import { useClipboard } from "../../hooks/useClipboard";
+import { useLocale } from "../../providers/AppLocaleProvider";
+import { Proposal } from "../../generated/graphql";
+
+const ProposalDescription: React.FC<{ proposal: Proposal }> = ({
+  proposal,
+}) => {
+  const copy = useClipboard();
+  const { translate } = useLocale();
+
+  const shareAPIAvailable = !!navigator.share;
```

Comment:
Shall we check with `navigator.canShare()`? Also please prevent using double exclamation marks as we are trying to enforce this rule in the future: https://github.com/oursky/eslint-oursky/issues/36

Reply:
If you are interested you can also make a `useShare` hook to encapsulate everything together

Reply:
Seems when share API is not supported, `navigator.canShare()` is also undefined.

<img width="259" alt="image" src="https://user-images.githubusercontent.com/106721852/172594939-bc8f11f4-3e10-4b2c-87c9-7c4114521221.png">